great_expectations/expectations/metrics/util.py

```diff
@@ -30,8 +30,8 @@
 )
 from great_expectations.compatibility.typing_extensions import override
 from great_expectations.execution_engine import (
-    PandasExecutionEngine,  # noqa: TCH001
-    SqlAlchemyExecutionEngine,  # noqa: TCH001
+    PandasExecutionEngine,
```

Comment:
looks like `static-analysis` is complaining. Should be pretty simple to move the import statement to a different block